src/schema/v2/profile.ts

```diff
@@ -35,7 +35,7 @@ export const ProfileType = new GraphQLObjectType<any, ResolverContext>({
     },
     href: {
       type: GraphQLString,
-      resolve: ({ id }) => `/${id}`,
+      resolve: ({ owner }) => `/partner/${owner.id}`,
```

Comment:
I think this change isn't correct (we can just remove it), since the `href` here should be for the profile, not the partner.

Reply:
Removed